### DISCUSSION TITLE
introduction/installation

### DISCUSSION THREAD

**Discussion by giscus** [ryan4yin/nixos-and-flakes-book#189] [Announcements]
# introduction/installation

An unofficial and opinionated book for beginners

https://nixos-and-flakes.thiscute.world/introduction/installation

**Comment by theinfinit** [comment]
For Windows users using WSL2, there is also [nix-community/NixOS-WSL](https://github.com/nix-community/NixOS-WSL). 

Instead of using, for example, Ubuntu with WSL2, you can use the NixOS. For me, this was a good and gentle way to familiarize myself with NixOS.
